blurb/blurb.py

```diff
@@ -749,6 +749,10 @@ def find_editor():
         editor = os.environ.get(var)
         if editor is not None:
             return editor
+    editor = subprocess.check_output(
+        ["git", "config", "--get", "core.editor"]).strip()
+    if editor:
+        return editor
```

Comment:
`git var GIT_EDITOR` is the magic incantation that we couldn't find when we were writing this.  +1 to switching to use that.